### ISSUE TITLE
Get object with  both region_id and region_name (and countries)

### ISSUE THREAD

**Issue by GMolini** [gehrj/vue-country-region-select#34]
It would be nice to have the option regionName (or countryName) also set the possibilty to return an object with both region_id and region_name (and countries)

**Comment by gehrj** [comment]
are you asking for something like this? @GMolini 

instead of returning: "Illinois"

it would return: { id: "IL", region_name: "Illinois" }

**Comment by GMolini** [comment]
yes! (sorry for the really late reply, the notification got buried under the rubble)
